citizen/citizen.c

```diff
@@ -295,12 +330,15 @@ void freeList(List *list) {
 }
 
 // search for a citizen in the list by a specific attribute and keyword.
-Citizen *searchCitizen(List *list, Type searchType, const char *keyword) {
+Citizen *searchCitizen(List *list, Type searchType, const char *keyword) 
```

Comment:
please follow the naming conventions on the variables, use snake_case

Reply:
still not followed

Reply:
Done, thanks.